python/tox.ini

```diff
@@ -15,33 +15,37 @@
 #
 
 [tox]
-envlist = py36, py37, py38, py39, flake8
+envlist = py36, py37, py38, py39, flake8, pylint
 
 [gh-actions]
 python =
     3.6: py36
     3.7: py37
-    3.8: py38, flake8
+    3.8: py38, flake8, pylint
     3.9: py39
 
 [testenv:flake8]
 basepython = python
 deps =
-    -r{toxinidir}/requirements_dev.txt
     -r{toxinidir}/requirements_lint.txt
-    -r{toxinidir}/requirements.txt
 commands =
     flake8 pynessie tests
     safety check --ignore=41002
     mypy --ignore-missing-imports -p pynessie
 
+[testenv:pylint]
+basepython = python
+deps =
+    -r{toxinidir}/requirements_lint.txt
+commands =
+    pylint pynessie
```

Comment:
why not lint tests too (in the spirit of "test is code")?

Reply:
good catch, will add that as well

Reply:
@dimas-b I have added pylint to the tests. However, I had to exchange the global variables with dataclass as pylint wasn't happy about using the global variables. Can you please check this out?